src/TreeViewList/example-items.json

```diff
@@ -5,18 +5,23 @@
     "tooltip": "Aerodynamics description goes here",
```

Comment:
I assume this tooltip is a placeholder and needs to be replaced.

Reply:
This is just an example